### PR TITLE
LEXI-89 Create ClayLabel

### PR DESCRIPTION
**Pull request by matuzalemsteles**

### CODE REVIEW

File: packages/clay-label/src/ClayLabel.soy [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	{else}
+		{$label}
+	{/if}
+
+	{if $closeable and $spritemap}
+		{call ClayButton.render}
+			{param ariaLabel: 'Close' /}
+			{param elementClasses: 'close' /}
+			{param events: [
+				'click': [
```
Comment (carloslancha) [inline packages/clay-label/src/ClayLabel.soy]: Why do you need to specify `button` as selector?

Comment (matuzalemsteles) [reply packages/clay-label/src/ClayLabel.soy]: You right, not necessary. Thx!

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
LGTM! 🎉

**Pull request merged**